app/templates/base.html

```diff
@@ -8,6 +8,7 @@
     <!-- 1. import css file here -->
 
```

Comment:
You are missing the code to actually import the CSS file
<link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">

Reply:
<link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">

Reply:
```<link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">```